rgw/v2/tests/s3_swift/test_versioning_with_objects.py

```diff
@@ -600,6 +600,31 @@ def test_exec(config, ssh_con):
                             bucket.name,
                             VERSIONING_STATUS["SUSPENDED"],
                         )
+                        if config.test_versioning_archive:
+                            arc_site_ip = utils.get_rgw_ip_arc()
+                            arc_site_ssh_con = utils.connect_remote(arc_site_ip)
+                            bucket_stats_op = utils.exec_shell_cmd(
```

Comment:
line 606 and 607 can be combined to single line:
bucket_stats_json_doc = json.loads(utils.exec_shell_cmd(f"radosgw-admin bucket stats --bucket {bucket_name_to_create}") )